frontend/packages/ceph-storage-plugin/src/features.ts

```diff
@@ -37,6 +37,9 @@ export const RGW_FLAG = 'RGW';
   Values should be formatted as OCS_<Console-Flag-Name> */
 export const OCS_SUPPORT_FLAGS = {
   MULTUS: 'OCS_MULTUS',
+  'FLEXIBLE-SCALING': 'OCS_FLEXIBLE_SCALING',
```

Comment:
```suggestion
  'FLEXIBLE_SCALING': 'OCS_FLEXIBLE_SCALING',
```
Using it will be much easier this way. I see this is the pattern anyway with ActionType constants.

Reply:
This is what as per the backend. I cannot understand, why we are puttng it in lowercase @bipuladh ?

Reply:
I would like to hard-code the keys as it is.

Reply:
lower case???